### ISSUE TITLE
Results pages should show commit positions, not git hashes

### ISSUE THREAD

**Issue by pcc** [catapult-project/catapult#4196]
1. Go to https://pinpoint-dot-chromeperf.appspot.com/results2/107dd144840000
2. Try to figure out whether each column is from before or after my change

== Expected behaviour ==

It should be easy to figure out which headers are before and which are after. Ideally each header should provide a commit position i.e. `chromium@531079`, `chromium@531026` etc instead of a hash. It might also be nice to provide some other indicator (such as colour coding) to easily determine before/after.

== Actual behaviour ==

I end up having to copy each hash (manually, because the page prevents copying the header columns) into a local command line to figure out what the commit position is and compare it against the commit position of my change.

**Comment by simonhatch** [comment]
@dave-2

**Comment by pcc** [comment]
Couple of problems that I see with that:
- it could result in ambiguity (I'm not thinking about especially prolific committers but rather things like autoroll bots which could more conceivably appear multiple times in a single bisect run)
- unless the columns are ordered by commit position, it still wouldn't really help you determine before/after (also, users would need to know that the columns are ordered as well as the direction in which they are ordered)

**Comment by dave-2** [comment]
That's true, the commit positions are useful for getting a sense of the ordering. I can change it to commit positions.

One note about autorolls: Pinpoint automatically expands DEPS rolls, and some dependent repos don't have commit positions, so you'd still see the git hash for those.
